xml/favicons.xml

```diff
@@ -2346,6 +2346,15 @@
     <param pos="0" name="service.product" value="R1Soft Server Backup Manager"/>
   </fingerprint>
 
+  <fingerprint pattern="^cc21dae362a981ea9ce93138cb855e66|372be2fa33d5836cdcf8c1867e6900c2|54bb0b4a353b89c42e63b8e8bd6ea504$">
```

Comment:
Please follow the pattern seen in other fingerprints that have multiple hashes so it correctly matches the values (`^(?:one|two|three)$`). The existing regex anchors only apply to the beginning of the first hash and the end of the third hash. For example, the existing fingerprint can incorrectly match the following values.
```
cc21dae362a981ea9ce93138cb855e66xyz
abc372be2fa33d5836cdcf8c1867e6900c2xyz
abc54bb0b4a353b89c42e63b8e8bd6ea504
```